tests/main/views/test_direct_award.py

```diff
@@ -64,6 +65,22 @@ def setup_class(cls):
         data_api_client.find_direct_award_projects = mock.Mock()
         data_api_client.find_direct_award_projects.return_value = cls._get_direct_award_project_list_fixture()
 
+    def test_renders_saved_search_overview(self):
+        self.login_as_buyer()
+        res = self.client.get(self.SAVE_SEARCH_OVERVIEW_URL)
+        assert res.status_code == 200
+
+        result_html = res.get_data(as_text=True)
+        assert "Your saved searches" in result_html
+        assert "Searching" in result_html
+        assert "Search ended" in result_html
```

Comment:
These tests are really forgiving and don't make it clear to the reader what's been tested. We should more definitively assert that we're checking a breadcrumb and two table headings.

Reply:
You're right, I'll have a look at making these better.